### PR TITLE
test: fix flaky explorer tests

### LINKED ISSUE #16330: Investigate Flaky Exlporer Tests
Query Builder Grouping (#16226) either introduced or exposed issues in the Explorer tests.

The issues were centered around the CodeMirror editor not loading properly or speedily enough, and the incorrect input being typed into the text area.

https://github.com/influxdata/influxdb/blob/master/ui/cypress/e2e/explorer.test.ts#L450

An errored out run (link may expire):
https://circleci.com/gh/influxdata/influxdb/78933?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

DataExplorer -- editing mode switching -- can switch to and from script editor mode (failed)

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Closes #16330

CodeMirror wraps the code textarea so it can apply eye candy to the text. Our tests were targeting the wrapped textarea and using the `force: true` flag to force cypress to type into it. This change instead targets the CodeMirror div that wraps the text area and doesn't force the typing. This seems to work as cypress waits for the text area to be typeable, which CodeMirror seems to handle.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -88,7 +88,7 @@
     "clean-webpack-plugin": "^3.0.0",
     "cross-env": "^5.2.0",
     "css-loader": "^3.1.0",
-    "cypress": "3.2.0",
+    "cypress": "^3.8.1",
```
Comment (hoorayimhelping) [inline ui/package.json]: this part of the diff should go away when https://github.com/influxdata/influxdb/pull/16401 gets merged.

File: ui/src/dataLoaders/components/verifyStep/DataListening.test.tsx [hunk 2]
```diff
@@ -12,7 +12,8 @@ describe('Onboarding.Components.DataListening', () => {
   describe('if button is clicked', () => {
     it('displays connection information', () => {
       const {getByTitle, getByText} = renderWithRedux(
-        <DataListening bucket="bucket" />)
+        <DataListening bucket="bucket" />
```
Comment (hoorayimhelping) [inline ui/src/dataLoaders/components/verifyStep/DataListening.test.tsx]: prettier /shrug

**Comment by hoorayimhelping** [comment]
@121watts been rerunning the tests in circle with no issue, can you show me how you make a loop? where do you put the loop?

File: ui/cypress/e2e/explorer.test.ts [hunk 3]
```diff
@@ -359,16 +358,16 @@ describe('DataExplorer', () => {
       cy.getByTestID('time-machine-submit-button').should('be.disabled')
 
       cy.getByTestID('flux-editor').within(() => {
-        cy.get('textarea').type('yo', {force: true})
+        cy.get('.CodeMirror').type('yo')
         cy.getByTestID('time-machine-submit-button').should('not.be.disabled')
       })
     })
 
     it('disables submit when a query is deleted', () => {
```
Comment (121watts) [inline ui/cypress/e2e/explorer.test.ts]:
```suggestion
for (let i = 0; i < 50; i++) {
    it('disables submit when a query is deleted', () => {
```

Comment (121watts) [reply ui/cypress/e2e/explorer.test.ts]: @hoorayimhelping here's an example of how to loop one test.

Comment (hoorayimhelping) [reply ui/cypress/e2e/explorer.test.ts]: sweet, thanks

**Pull request merged**